engine/src/main/java/org/destinationsol/SolApplication.java

```diff
@@ -175,6 +175,10 @@ public void create() {
 
     @Override
     public void resize(int newWidth, int newHeight) {
+        //To prevent application crashing, dont resize the height and width to 0 s, this condition checks it
+        if(newWidth==0 && newHeight==0){
```

Comment:
Please be sure you have the Checkstyle plugin enabled in your IDE and look for formatting warnings - this isn't formatted right when it comes to whitespace :-)

Reply:
oh actually I just ignored the checkstyle errors but now it seems to be okay